### PR TITLE
Add matlab

### PR DESCRIPTION
**Pull request by himanshu1603**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:
It is used in many repositories

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by himanshu1603** [comment]
@defunkt   please review my PR.

**Comment by MikeMcQuaid** [comment]
@himanshu1603 Please don't CC people by name, we will get to your PRs, thanks!

File: topics/matlab/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+created_by: Cleve Moler
+display_name: MATLAB
+logo: matlab.png
+released: 1984
+short_description: MATLAB is a programming platform for numerical computation and visualization.
+topic: matlab
+url: http://mathworks.com/products/matlab
+wikipedia_url: https://en.wikipedia.org/wiki/MATLAB
+---
+MATLAB is a high-performance language developed by MathWorks for technical computing, visualization, and programming. It is written in C, C++, Java and available for the various operating system such as Windows, macOS, and Linux.Generally used by engineers and scientists in industry and academics for data analysis, signal processing, optimization and many other types of scientific computations.
```
Comment (MikeMcQuaid) [inline topics/matlab/index.md]: - `for the various operating system such as Windows, macOS, and Linux` change to just `for Windows, macOS, and Linux`
- `Linux.Generally` can you stick a space after `.`.

Thanks!

**Comment by MikeMcQuaid** [comment]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @himanshu1603!

**Comment by himanshu1603** [comment]
Thanks!

**Pull request merged**